### PR TITLE
[v5.0.6] Update installer.iss with correct version and copyright dates

### PR DESCRIPTION
**Pull request by krzywon**
## Description
This updates the version and copyright date in installer.iss for v5.0.6.

## Review Checklist (please remove items if they don't apply):

- [ ] Code has been reviewed
- [ ] Functionality has been tested
- [ ] Windows installer (GH artifact) has been tested (installed and worked) 
- [ ] MacOSX installer (GH artifact) has been tested (installed and worked) 
- [ ] User documentation is available and complete (if required)
- [ ] Developers documentation is available and complete (if required)
- [ ] The introduced changes comply with SasView license (BSD 3-Clause)

### CODE REVIEW

**Comment by butlerpd** [comment]
I think it is too late for b1. The question however might be whether to change this to 5.0.6 (for all betas even) on the grounds that the final 5.0.6 will just be in the same directory?  When launching the version number is now prominently displayed in the top left corner of the SasView application window.  This would make releases a bit easier ... and less worry that the final release will default to 5.0.6bx instead of 5.0.6?

File: installers/installer.iss [hunk 1]
```diff
@@ -2,8 +2,8 @@
 ; SEE THE DOCUMENTATION FOR DETAILS ON CREATING INNO SETUP SCRIPT FILES!
 
 #define MyAppName "SasView"
-#define MyAppVersion "5.0.5"
-#define MyAppPublisher "(c) 2009 - 2021, UTK, UMD, NIST, ORNL, ISIS, ESS, ILL, ANSTO, TU Delft and DLS"
+#define MyAppVersion "5.0.6b1"
```
Comment (butlerpd) [inline installers/installer.iss:5]: For the installer we should just use 5.0.6 (for the betas and the final release).  Betas should be overwritten by the final release in any case.  Once that is changed this should be ready to merge.

**Review by butlerpd — CHANGES_REQUESTED** [review]

**Comment by butlerpd** [comment]
also .. why is it failing ubuntu unit tests?

**Comment by krzywon** [comment]
> also .. why is it failing ubuntu unit tests?

The ubuntu failures were due to intermittent connection issues with azure.

**Pull request merged**